### PR TITLE
fix(LoginPage): small fixes.

### PR DESCRIPTION
**Pull request by Laviro**
While trying to implement the [LoginPage component in Foreman](https://github.com/theforeman/foreman/pull/6234)
I faced some problems which are fixed in this PR.

### CODE REVIEW

File: packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardRememberMe.js [hunk 1]
```diff
@@ -15,7 +16,7 @@ LoginCardRememberMe.propTypes = {
 };
 
 LoginCardRememberMe.defaultProps = {
-  label: 'Keep me logged in for 30 days',
+  label: '',
```
Comment (jeff-phillips-18) [inline packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardRememberMe.js]: not sure about defaulting to text that is not handled by default. There is nothing here that will expire login after 30 days, correct?

File: packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardForm.js [hunk 2]
```diff
@@ -16,15 +16,23 @@ const LoginCardForm = ({
   forgotPassword,
   rememberMe,
   submitError,
-  showError
+  showError,
+  attributes
```
Comment (jeff-phillips-18) [inline packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardForm.js]: we usually just use `...props` and pass those along. no need to set a propType for this.

File: packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardRememberMe.js (hunk 1 above)
Comment (jeff-phillips-18) [reply packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardRememberMe.js]: I think null is better so this doesn't show up inadvertently.

Comment (Laviro) [reply packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardRememberMe.js]: done

**Pull request merged**